apps/api/pecanapi-spec.yml

```diff
@@ -335,29 +329,113 @@ paths:
     post:
       tags:
         - workflows
-        - 
       summary: Submit a new PEcAn workflow
       requestBody:
         required: true
         content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/Workflow'
+          # application/json:
+          #   schema:
+          #     $ref: '#/components/schemas/Workflow'
           application/xml:
             schema:
-              $ref: '#/components/schemas/Workflow'
+              xml:
```

Comment:
it is a good idea to have these schema's defined at the bottom of the swagger file. This way you can reference them in multiple places. (same goes for responses).

Reply:
Yeah...I will do that...coz the same schema will be repeated for the `application/json` as well